dev-util/gengetopt/gengetopt-2.23.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A tool for generating scanners"
+DESCRIPTION="Gengetopt is a tool to write command \
+line option parsing code for C programs."
+HOMEPAGE="https://www.gnu.org/software/gengetopt/"
+COPYRIGHT="2001-2012 Lorenzo Bettini
+ 	2019 Tomas Volf
+	"
+LICENSE="GNU GPL v3"	
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/gengetopt/gengetopt-$portVersion.tar.xz"
+CHECKSUM_SHA256="b941aec9011864978dd7fdeb052b1943535824169d2aa2b0e7eae9ab807584ac"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
```

Comment:
If this works for gcc2 you can remove $secondaryArchSuffix, otherwise you need to add a line for secondary architecture support here